en/asgardeo/docs/guides/api-security/aws/securing-aws-http-apis.md

```diff
@@ -0,0 +1,279 @@
+---
+template: templates/quick-start.html
+heading: Secure a Pharmacy User API with AWS API Gateway and Asgardeo
+description: This guide walks you through securing a Pharmacy User API using AWS API Gateway and WSO2 Asgardeo. You'll deploy a Lambda function, expose it via an HTTP API, and protect the endpoint using JWT validation powered by Asgardeo.
```

Comment:
```suggestion
description: This guide walks you through securing a Pharmacy User API using AWS API Gateway and Asgardeo. You'll deploy a Lambda function, expose it via an HTTP API, and protect the endpoint using JWT validation powered by Asgardeo.
```

JWT validation is independent of Asgardeo, isn't it? in that case, "JWT validation powered by Asgardeo." isn't accurate, IMO

Reply:
Addressed in [e4fc8aa](https://github.com/wso2/docs-is/pull/5253/commits/e4fc8aa8a7f3140b9f3447455ff995b2ad2ef2ac)